go/head.templ

```diff
@@ -5,7 +5,7 @@ templ DefaultCSS() {
 	<link rel="preconnect" href="https://fonts.googleapis.com"/>
 	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin/>
 	<link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&display=swap" rel="stylesheet"/>
-	<link href="https://cdn.jsdelivr.net/npm/@invopop/popui@0.0.49/dist/popui.css" rel="stylesheet"/>
+	<link href="https://cdn.jsdelivr.net/npm/@invopop/popui@0.0.50/dist/popui.css" rel="stylesheet"/>
```

Comment:
This feels a bit risky, as it implies you need to have a release before you can make the release... that's very strange. At the very least you want to use some kind of variable so that you're not manually updating the styles, but it might also make sense to use the Go "embed" feature to service the styles directly. The version control stuff I applied to access could be useful here if its possible to generate the CSS in advance: https://github.com/invopop/access/blob/e6756f8e2dec1cb9054a439e7171c871ace56e00/internal/interfaces/web/assets/assets.go